src/core/tests/pass/serialization/serialize.cpp

```diff
@@ -98,7 +98,8 @@ INSTANTIATE_TEST_SUITE_P(
                     std::make_tuple("loop_2d_add.xml", "loop_2d_add.bin"),
                     std::make_tuple("nms5_dynamism.xml", "nms5_dynamism.bin"),
                     std::make_tuple("if_diff_case.xml", "if_diff_case.bin"),
-                    std::make_tuple("if_body_without_parameters.xml", "if_body_without_parameters.bin")));
+                    std::make_tuple("if_body_without_parameters.xml", "if_body_without_parameters.bin"),
+                    std::make_tuple("string_parameter.xml", "string_parameter.bin")));
```

Comment:
should we also test serialize of string parameter?

Reply:
Do we have test suite for such purpose? Please advice

Reply:
Discussed offline, figured out that serialization is also tested